pymeasure/instruments/rohdeschwarz/sfm.py

```diff
@@ -0,0 +1,1501 @@
+
+        valid range: 0 .. 0.9
+        """,
+        validator = strict_range,
+        values=[0, 0.9],
+        )
+
+    coder_pilot_frequency = Instrument.control(
+        "SOUR:TEL:MOD:COD:PIL:FREQ?",
+        "SOUR:TEL:MOD:COD:PIL:FREQ %d",
+        """ A int property that controls the pilot frequency of the coder
+
+        valid range: 40 .. 60 kHz
+        """,
+        validator = strict_range,
+        values=[5E4, 6E4],
+        )
+
+    coder_pilot_deviation = Instrument.control(
+        "SOUR:TEL:MOD:COD:PIL:FREQ:DEV?",
+        "SOUR:TEL:MOD:COD:PIL:FREQ:DEV %d",
+        """ A int property that controls deviation of the pilot frequency of the coder
+
+        valid range: 1 .. 4 kHz
+        """,
+        validator = strict_range,
+        values=[1E3, 4E3],
+        )
+
+    #External modulation (3.6.6.5)
+    #TODO (for all in ext mode MOD-->MOD[1..6])
+    external_modulation_power = Instrument.control(
+        "SOUR:TEL:MOD:EXT:POW?",
+        "SOUR:TEL:MOD:EXT:POW %d",
+        """ A int property that controls the setting for the external modulator output power
+
+        valid range: -7..0 dBm
+        """,
+        validator = strict_range,
+        values=[-7, 0],
+        )
+
+    external_modulation_frequency = Instrument.control(
+        "SOUR:TEL:MOD:EXT:FREQ?",
+        "SOUR:TEL:MOD:EXT:FREQ %d",
+        """ A int property that controls the setting for the external modulator output power
+
+        valid range: 32 .. 46 MHz
+        """,
+        validator = strict_range,
+        values=[32e6, 46e6],
+        )
+
+    #NICAM system (3.6.6.6)
+    #TODO (for all in NICAM MOD-->MOD[1..6])
+    nicam_mode = Instrument.control(
+        "SOUR:TEL:MOD:NIC:AUD:MODE?",
+        "SOUR:TEL:MOD:NIC:AUD:MODE %s",
+        """ A string property that controls the signal type to be sent via NICAM
+
+        Possible values are:
+
+        ======  =======
+        Value   Meaning
+        ======  =======
+        MON     Mono sound + NICAM data
+        STER    Stereo sound
+        DUAL    Dual channel sound
+        DATA    NICAM data only
+        ======  =======
+
+        """,
+        validator = strict_discrete_set,
+        values=["MON","STER","DUAL","DATA"],
+        )
+
+    nicam_audio_frequency = Instrument.control(
+        "SOUR:TEL:MOD:NIC:AUD:FREQ?",
+        "SOUR:TEL:MOD:NIC:AUD:FREQ %d",
+        """ A int property that controls the frequency of the internal sound generator
+
+        valid range: 0 Hz .. 15 kHz
+        """,
+        validator = strict_range,
+        values=[0, 1.5E4],
+        )
+
+    nicam_preemphasis_enabled = Instrument.control(
+        "SOUR:TEL:MOD:NIC:AUD:PRE?",
+        "SOUR:TEL:MOD:NIC:AUD:PRE %d",
+        """ A bool property that controls the status of the J17 preemphasis,
+
+        ======  =======
+        Value   Meaning
+        ======  =======
+        False   preemphasis disabled
+        True    preemphasis enabled
```

Comment:
I don't think you need the table, the name is clear!

Reply:
yes,
table removed